docs/data/xrootd/xrootd-authorization.md

```diff
@@ -1,36 +1,185 @@
+[authorization database](#authorization-database) is used to provide fine-grained file access.
 
 !!! note
-    On the data nodes, the files will actually be owned by Unix user `xrootd` (or other daemon user), not as the user
+    On data nodes, files will be owned by Unix user `xrootd` (or other daemon user), not as the user
     authenticated to, under most circumstances.
     XRootD will verify the permissions and authorization based on the user that the security plugin authenticates you
     to, but, internally, the data node files will be owned by the `xrootd` user. If this behaviour is not desired, enable
     [XRootD multi-user support](install-standalone.md#enabling-multi-user-support). 
 
-#### Authorization file
+Authorizing Bearer Tokens
+-------------------------
+
+The OSG configurations of XRootD support authorization of bearer tokens such as macaroons, SciTokens, or WLCG tokens.
+Encoded in the bearer tokens themselves are information about the files that they should have read/write access to and
+in the case of SciTokens and WLCG tokens, you may configure XRootD to further restrict access.
+
+### Configuring SciTokens/WLCG Tokens ###
+
+SciTokens and WLCG Tokens are asymmetrically signed bearer tokens: they are signed by a token issuer (e.g., CILogon, IAM)
+and can be authenticated with the token issuer's public key.
+To configure XRootD to accept tokens from a given token issuer use the following instructions:
+
+1.  Add a section for each token issuer to `/etc/xrootd/scitokens.conf`:
+
+        [Issuer <NAME>]
+        issuer = <URL>
+        base_path = <RELATIVE PATH>
+
+    Replacing `<NAME`> with a descriptive name, `<URL>` with the token issuer URL, and `base_path` to a path relative to
+    [`rootdir`](install-standalone.md#configuring-xrootd) that the client should be restricted to accessing.
+
+1.  **(Optional)** if you want to map the incoming token for a given issuer to a Unix username:
+
+    1.  Install [xrootd-multiuser](install-standalone.md#enabling-multi-user-support)
+
+    1.  Add the following to the relevant issuer section in `/etc/xrootd/scitokens.conf`:
+
+            map_subject = True
+
+1.  **(Optional)** if you want to only accept tokens with the appropriate `aud` field, the following to
+    `/etc/xrootd/scitokens.conf`:
+
+        [Global]
+        audience = <COMMMA SEPARATED LIST OF AUDIENCES>
+
+An example configuration that supports the OSG Connect and CMS:
+
+```
+[Global]
+audience = https://testserver.example.com/, MySite
+
+[Issuer OSG-Connect]
+
+issuer = https://scitokens.org/osg-connect
+base_path = /stash
+map_subject = True
+
+[Issuer CMS]
+
+issuer = https://scitokens.org/cms
+base_path = /user/cms
+```
+
+### Configuring macaroons ###
+
+Macaroons are symetrically signed bearer tokens so your XRootD host must have access to the same secret key that is used
+to sign incoming macaroons.
```

Comment:
```suggestion
Macaroons are symmetrically signed bearer tokens so XRootD both issues and verifies its access tokens using the same secret key. When used in an XRootD cluster, all Data Nodes and the Redirector need access to the same secret
```

Reply:
```suggestion
Macaroons are symetrically signed bearer tokens so your XRootD host must have access to the same secret key that is used
to sign incoming macaroons.
When used in an XRootD cluster, all Data Nodes and the Redirector need access to the same secret.
```

Adding a missing period + moving prose to a new line for cleaner diffs in the future